tensorflow_datasets/image/imagenette.py

```diff
@@ -0,0 +1,139 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Imagenette: a subset of 10 easily classified classes from Imagenet (tench,
+English springer, cassette player, chain saw, church, French horn, garbage
+truck, gas pump, golf ball, parachute)"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import io
+import itertools
+import os
+
+import numpy as np
+import tensorflow as tf
+
+import tensorflow_datasets.public_api as tfds
+
+# TODO(imagenette): BibTeX citation
+_CITATION = """
+"""
+
+_DESCRIPTION = """\
+Imagenette is a subset of 10 easily classified classes from the Imagenet
+dataset. It was originally prepared by Jeremy Howard of FastAI. The objective
+behind putting together a small version of the Imagenet dataset was mainly
+because running new ideas/algorithms/experiments on the whole Imagenet take a
+lot of time.
+
+This version of the dataset allows researchers/practitioners to quickly try out
+ideas and share with others. The dataset comes in three variants:
+
+  * Full size
+  * 320 px
+  * 160 px
+This dataset consists of the Imagenette dataset {size} variant.
+"""
+
+_LABELS_FNAME = 'image/imagenette_labels.txt'
+_URL_PREFIX = 'https://s3.amazonaws.com/fast-ai-imageclas'
+
+
+class ImagenetteConfig(tfds.core.BuilderConfig):
+  """BuilderConfig for Imagenette."""
+
+  def __init__(self, size, **kwargs):
+    super(ImagenetteConfig, self).__init__(
+        version=tfds.core.Version('0.1.0'), **kwargs)
+    self.size = size
+
+
+def _make_builder_configs():
+  configs = []
+  for size in ['full-size', '320 px', '160 px']:
+    configs.append(
+        ImagenetteConfig(
+            name='%s' % (size),
+            size=size,
+            description=_DESCRIPTION.format(size=size)))
+  return configs
+
+
+class Imagenette(tfds.core.GeneratorBasedBuilder):
+  """Imagenette:A smaller subset of 10 easily classified classes from
```

Comment:
please run the lint on your code:

https://github.com/tensorflow/datasets/blob/38189d93462b6e4f8fab820d416d8959c11d85e4/docs/add_dataset.md#5-check-your-code-style

Reply:
Okay, I will do it.

Reply:
Did it run ? (as normally it complains that the comment should fit in a single line (with a "dot" at the end).

Reply:
I haven't run it on the whole tensorflow_dataset/image directory but I have checked it with the files that I have added and it is given 10/10 on both the files that I have added.

Reply:
![image](https://user-images.githubusercontent.com/44494703/70358259-ee4acb80-189e-11ea-990e-8d61f5439ebd.png)